src/ansys/fluent/core/services/datamodel_tui.py

```diff
@@ -192,7 +192,8 @@ def execute(self, *args, **kwargs) -> Any:
         if self._path.startswith("/query/"):
             return self._execute_query(request)
         else:
-            logger.debug(f"TUI Command: {request}")
+            request_str = " ".join(str(request).split())
```

Comment:
What's going on here? Is it just to minimise whitespace?

Reply:
`.split()` removes whitespaces and line breaks

Reply:
See "before" and "after" for "TUI calls" in the original post above to see what that does